File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        count = 0
        for i in range(len(s)-1, -1, -1):
            if s[i].isalpha():
                count += 1 
            if s[i] == " " and count != 0:
                break
        return count
        # spl = s.split()
        # return len(spl[-1])

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int count = 0;
        for (int i=s.length()-1; i >= 0; i--){
            char ele = s.charAt(i);
            if (Character.isLetter(ele)){
                count++;
            }
            else if (ele == ' ' && count != 0){
                break;
            }
        }
        return count;
    }
}